trino-s3-proxy/src/test/java/io/trino/s3/proxy/server/testing/TestingTrinoS3ProxyServerModule.java

```diff
@@ -30,5 +33,8 @@ protected void moduleSpecificBinding(Binder binder)
 
         binder.bind(RemoteS3Facade.class).to(TestingRemoteS3Facade.class).in(Scopes.SINGLETON);
         binder.bind(TestingRemoteS3Facade.class).in(Scopes.SINGLETON);
+
+        newOptionalBinder(binder, StsController.class).setBinding().to(TestingStsController.class).in(Scopes.SINGLETON);
```

Comment:
can we also remove this optional binding?

Reply:
I thought I did - rebasing issue. 1 sec...

Reply:
done